style/style.css

```diff
@@ -0,0 +1,312 @@
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+}
+
+body{
+    background-color: #ffffff;
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-template-rows: 200px auto auto;
+    grid-template-areas: 
+    "hd"
+    "section"
+    "footer";
+    gap: 1rem;
+}
+
+
+.header{
+    /* background-color: red; */
+    grid-area: hd;
+    background-image: url("/img/header\ pic.avif");
+    object-fit: cover;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    width: 100%;
+}
+
+.nav-header {
+    display: none;
+}
+
+.add {
+    max-width: 24rem;
+    word-wrap: break-word;
```

Comment:
I don't think you need this; it will cause fragments of words to break to the next line. Best for readability to let words stay whole and wrap using the whitespace between them.